src/features/offer/components/OfferTile/useHandleOfferTile.ts

```diff
@@ -0,0 +1,20 @@
+import { analytics } from 'libs/analytics'
+import { PartialOffer, usePrePopulateOffer } from 'shared/offer/usePrePopulateOffer'
+
+type AnalyticsParams = Parameters<typeof analytics.logConsultOffer>[0]
+
+export type HandlePressOfferParams = {
+  offer: PartialOffer
+  analyticsParams: AnalyticsParams
+}
+
+export const useHandleOfferTile = () => {
+  const prePopulateOffer = usePrePopulateOffer()
+  function handlePressOffer({ offer, analyticsParams }: HandlePressOfferParams) {
+    // We pre-populate the query-cache with the data from the search result for a smooth transition
```

Comment:
il me semble que ce hook est aussi utilisé en dehors de la recherche

```suggestion
    // We pre-populate the query-cache with the data from the previous query for a smooth transition
```